extensions/database.py

```
@@ -13,41 +13,37 @@
-        "UpvoteFeatures": [int],
-        "DownvoteFeatures": [int],
-        "CreateSessions": [int, str],
-        "GetPrivateSessions": [int],
-        "ValidateSessions": [bytes, str],
-        "CreateUsers": [str, str, str],
-        "GetPrivateUsers": [int],
-        "GetUidUsers": [int],
-        "GetUnameUsers": [str]
-    }
     # CLASS -> INIT
     async def init(self, socket: æ.WebSocket) -> namespace:
```

Ruff (0.12.2)

Unused method argument: `socket`

(ARG002)